### PR TITLE
tlp: add zzz resume/suspend hooks

### PR DESCRIPTION
**Pull request by hazen2215**
https://github.com/linrunner/TLP/issues/656

also make it conflict with perl-Unicode-Tussle (/usr/bin/nfc)

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by nbelikov** [comment]
A small suggestion, if I may:

    exec tlp {suspend,resume}

`exec` isn't necessary but would improve performance a little bit essentially for free.

I use the same set of scripts myself (which only differ in aforementioned exec's) and they work as expected.

**Comment by hazen2215** [comment]
added `exec`.

**Comment by hazen2215** [comment]
modified so the hooks will execute `tlp {suspend,resume}` only if `tlp` service is enabled

**Comment by leahneukirchen** [comment]
cc @adbrown101

**Comment by dkwo** [comment]
what about https://github.com/void-linux/void-packages/issues/40739 ?

File: srcpkgs/tlp/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ changelog="https://raw.githubusercontent.com/linrunner/TLP/main/changelog"
 distfiles="https://github.com/linrunner/TLP/archive/${version}.tar.gz"
 checksum=b5f08f00d535c26abc49b336b4c7264c6e5fb7bc3de8054eaabeebdd00e0760e
 
-conflicts="laptop-mode>=0"
+conflicts="laptop-mode>=0 perl-Unicode-Tussle>=0"
```
Comment (sgn) [inline srcpkgs/tlp/template:14]: Maybe we should rename `nfc` to `tlp-nfc` instead?

**Pull request merged**